react/EditorProvider.js

```diff
@@ -88,17 +100,22 @@ class EditorProvider extends Component {
   }
 
   componentDidMount() {
-    const { children, page } = this.props
+    const { page } = this.props
     const root = page.split('/')[0]
     if (root !== 'admin') {
-      document.getElementById('render-container') && document.getElementById('render-container').classList.add('editor-provider')
+      document.getElementById('render-container') &&
+        document
+          .getElementById('render-container')
+          .classList.add('editor-provider')
```

Comment:
Hint, cache value to prevent double query in the DOM:
```
var x = document.getElementById('render-container')
if (x) { x.classList.add('editor-provider') } 
```
or similar.